src/reaper_osara.cpp

```diff
@@ -3956,6 +3956,37 @@ void cmdManageTempoTimeSigMarkers(Command* command) {
 	Main_OnCommand(40256, 0); // Tempo envelope: Insert tempo/time signature change marker at edit cursor...
 }
 
+void cmdSelectItemsUnderEditCursorOnSelectedTracks(Command* command) {
+	vector<MediaItem*> items;
+	for (int i = 0; i < CountTracks(nullptr); ++i) {
+		MediaTrack* track = GetTrack(nullptr, i);
+		for (int j = 0; j < CountTrackMediaItems(track); ++j) {
+			MediaItem* item = GetTrackMediaItem(track, j);
+			items.push_back(item);
+		}
+	}
+	PreventUIRefresh(1); // SWS use this
+	Main_OnCommand(40289,0); // unselect all items
+	int i;
+	double cursorPosition=GetCursorPosition();
+	for (i=0;i<(int)items.size();++i) {
+		MediaTrack* track = (MediaTrack*)GetSetMediaItemInfo(items[i],"P_TRACK", 0);
+		if (isTrackSelected(track)) {
+			double itemStart = *(double*)GetSetMediaItemInfo(items[i], "D_POSITION", 0);
+			double itemLength = *(double*)GetSetMediaItemInfo(items[i], "D_LENGTH", 0);
+			double itemEnd = itemStart + itemLength;
+			if (cursorPosition >= itemStart && cursorPosition <= itemEnd) {
+				bool makeSelected = true;
+				GetSetMediaItemInfo(items[i], "B_UISEL", &makeSelected);
```

Comment:
You can use `&bTrue` here rather than defining your own bool. (This is defined in osara.h for this purpose.)